code/game/objects/items/weapons/grenades/stinger.dm

```diff
@@ -0,0 +1,62 @@
+/proc/stinger(var/source,var/fragx,var/fragy,var/light_dam,var/flash_dam,var/p_dam,var/p_range,var/can_cover=TRUE,var/shard_range = 50)
```

Comment:
This has no reason to be a global proc, repath it under /obj/item/grenade/stinger and properly space the parameters it takes

`fragx` and `fragy` doesn't seem to be used as coordinates but as bounds for how many fragments to launch, they should therefore be called `fragments_min` and `fragments_max` or similarly